src/ui/server/controllers/insurance/business/broker/index.test.ts

```diff
@@ -7,8 +7,14 @@ import insuranceCorePageVariables from '../../../../helpers/page-variables/core/
 import mapApplicationToFormFields from '../../../../helpers/mappings/map-application-to-form-fields';
 import { mockReq, mockRes, mockApplication } from '../../../../test-mocks';
 import generateValidationErrors from './validation';
+import { FIELDS } from '../../../../content-strings/fields/insurance/your-business';
 
-const { USING_BROKER } = FIELD_IDS.BROKER;
+const { BROKER: BROKER_FIELDS } = FIELDS;
+
+// below will be removed on next PR, linting not working for this line
+// eslint-disable-next-line operator-linebreak
+const { USING_BROKER, BROKER_HEADING, BROKER_NAME, BROKER_ADDRESS_LINE_1, BROKER_ADDRESS_LINE_2, BROKER_COUNTY, BROKER_TOWN, BROKER_POSTCODE } =
```

Comment:
Can just add new line break to each destructure?

Reply:
eslint complains about that too

Reply:
Interesting - I think if we rename like above comment so we don't have `BROKER_` prefix, eslint will be OK?